report_templates/job_invocation_report.erb

```diff
@@ -0,0 +1,26 @@
+<%#
+name: Job Invocation report
+snippet: false
+template_inputs:
+- name: job_id
+  required: false
+  input_type: user
+  description: ID of job to report
+  advanced: false
+  value_type: plain
+  resource_type: JobInvocation
+model: ReportTemplate
+require:
+- plugin: foreman_remote_execution
+  version: '2.0.3'
+-%>
+<%- invocation = load_job_invocation_by_id(input('job_id')) -%>
+<%- invocation.template_invocations_hosts.each do |host| -%>
+<%- sub_task_output = invocation.sub_task_for_host(host).output_by_type -%>
+<%-   report_row({ host: host.name,
```

Comment:
you can omit `{}` and if you put `host: host.name` on the new line after opening bracket, it will be simpler to comment out for people, who want to customize this report